server/server.go

```diff
@@ -0,0 +1,61 @@
+package server
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/brave-intl/bat-go/middleware"
+	"github.com/brave/go-translate/controller"
+	"github.com/getsentry/raven-go"
+	"github.com/go-chi/chi"
+	chiware "github.com/go-chi/chi/middleware"
+	"github.com/pressly/lg"
+	"github.com/sirupsen/logrus"
+)
+
+func setupLogger(ctx context.Context) (context.Context, *logrus.Logger) {
+	logger := logrus.New()
+	// Redirect output from the standard logging package "log"
+	lg.RedirectStdlogOutput(logger)
+	lg.DefaultLogger = logger
+	ctx = lg.WithLoggerContext(ctx, logger)
+	return ctx, logger
+}
+
+func setupRouter(ctx context.Context, logger *logrus.Logger) (context.Context, *chi.Mux) {
+	r := chi.NewRouter()
+
+	r.Use(chiware.RequestID)
+	r.Use(chiware.RealIP)
+	r.Use(chiware.Heartbeat("/"))
+	r.Use(chiware.Timeout(60 * time.Second))
+	r.Use(middleware.BearerToken)
+
+	if logger != nil {
+		// Also handles panic recovery
+		r.Use(middleware.RequestLogger(logger))
+	}
+
+	r.Mount("/", controller.TranslateRouter())
+	r.Get("/metrics", middleware.Metrics())
+
+	return ctx, r
+}
+
+// StartServer starts the translate proxy server on port 8195
+func StartServer() {
+	serverCtx, logger := setupLogger(context.Background())
+	logger.WithFields(logrus.Fields{"prefix": "main"}).Info("Starting server")
```

Comment:
We need to make sure that we are not logging any user identifiable data here.

Reply:
<img width="1283" alt="Screen Shot 2019-03-20 at 2 33 00 PM" src="https://user-images.githubusercontent.com/4730197/54720624-1631ad80-4b1d-11e9-9e25-26ac4ea32b75.png">
This shares the same pattern as go-update so https://github.com/brave/go-update/issues/22 applies here too.
As @jumde suggested, we should have a log-retention policy enforced on the infra side when we deploy the server.

cc @tomlowenthal